frontend_lib/src/component/Avatar/Avatar.jsx

```diff
@@ -37,7 +37,6 @@ export class Avatar extends React.Component {
         <img
           className='avatar'
           src={`${avatarBaseUrl}/preview/jpg/${sizeAsNumber}x${sizeAsNumber}/${filenameInUrl}`}
-          alt={props.t('Avatar of {{publicName}}', { publicName })}
```

Comment:
I'm expecting this change comes from our recent discussion about the alt attribute.

After writing a bit of guideline in this comment, then thinking back about it, I decided to ctrl+t a bit about "alt".
W3C says its required. MDN says it isn't required but "incredibly useful for accessibility".
Which means I was wrong, it indeed is useful for accessibility.

So... maybe "Avatar of {{publicName}}" is actually the best

Reply:
Well, I've read this while I was removing the alt... I feel like the Avatar displayed here is purely decorative. Like we know that the line where there will be the avatar, there will be the user name too... 
Or am I wrong?

Looks like in any case, the `alt` should be there, but the value should be empty.

Reply:
I agree, `alt=""` is the best here.
I found a good article aboud alt: https://webaim.org/techniques/alttext/

Reply:
Solved in [41219fb](https://github.com/tracim/tracim/pull/6146/commits/41219fbb58e78742229acde6c48d16ae0f65bafa)